### DISCUSSION TITLE
Why did you all adopt MIT and not another software license?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#358] [Q&A] [answered]
Hello, folks!

I would like to know if there is any special reason to prefer the MIT license than another type of license. Why did you all adopt MIT and not another software license? what are the pros and cons of the MIT license?

**Comment by viliusle** [comment, marked as answer]
I had another free license in past, it was probably GPL and for me it created so much troubles. It got lots of questions like "am I allowed to to X" and so on, and I had to spend lots of time researching on these things. 

Once I switched to MIT, it was like heaven. Basically 0 troubles. 
p.s. MIT basically states that you have no restrictions.
